packages/contentstack-migration/package.json

```diff
@@ -4,7 +4,7 @@
   "author": "@contentstack",
```

Comment:
@abhinav-from-contentstack This version of `@contentstack/cli-migration` plugin is bumped twice from 1.1.0 to 1.1.2.
This should be 1.1.1 only.